libs/ui/packages/crypto-icons/package.json

```diff
@@ -26,7 +26,7 @@
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "node scripts/buildReactIcons && tsc --noEmit false && node scripts/transpile",
+    "build": "pnpm run clean && node scripts/buildReactIcons && tsc --noEmit false && node scripts/transpile",
```

Comment:
why this change here?

Reply:
It was removed by error bi DX team lately, and without it you can't build locally. That change is also included in the Typescript PR